CONTRIBUTING.md

```diff
@@ -0,0 +1,86 @@
+
+- Take a look at the Existing [Issues](https://github.com/eddiejaoude/stargate) or create your own Issues!
+- Wait for the Issue to be assigned to you after which you can start working on it.
+- Fork the Repo and create a Branch for any Issue that you are working upon.
+- Create a Pull Request which will be promptly reviewed and suggestions would be added to improve it.
+- Add Screenshots to help us know what this Script is all about.
+
+
+## ⭐HOW TO MAKE A PULL REQUEST:
+
+**1.** Fork [this](https://github.com/eddiejaoude/stargate) repository.
+Click on the <a href="https://github.com/eddiejaoude/stargate"><img src="https://img.icons8.com/ios/24/000000/code-fork.png"></a> symbol at the top right corner.
+
+**2.** Clone the forked repository.
+
+```bash
+git clone https://github.com/<your-github-username>/stargate
+```
+
+**3.** Navigate to the project directory.
+
+```bash
+cd stargate
+```
+**5.** Create a new branch:
+```bash
+git checkout -b <YourBranchName>
+```
+
+**6.** Make changes in source code.
+
+**7.** Stage your changes and commit
+
+```bash
+git add <file1,file2....>
+```
+```bash
+git commit -m "<your_commit_message>"
+```
+
+**8.** Push your local commits to the remote repo.
+
+```bash
+git push origin <YourBranchName>
+```
+
+**9.** Create a [PR](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request) !
+
+**10.** **Congratulations!** Sit and relax, you've made your contribution to [Stargate](https://github.com/eddiejaoude/stargate) project.
+
+***:trophy: After this, project leaders and mentors will review the changes and will merge your PR if they are found good, otherwise we will suggest the required changes.***
+
+## Style Guides for Git Commit Messages:memo:
+
+#### Here's a list of some good to have points, that can add more value to your contribution logs.
+
+- fix: a commit of the type fix patches a bug in your codebase (this correlates with PATCH in semantic versioning).
+- feat: a commit of the type feat introduces a new feature to the codebase (this correlates with MINOR in semantic versioning).
+- BREAKING CHANGE: a commit that has a footer BREAKING CHANGE:, or appends a ! after the type/scope, introduces a breaking API change (correlating with MAJOR in semantic           versioning). A BREAKING CHANGE can be part of commits of any type.
+- types other than fix: and feat: are allowed, for example @commitlint/config-conventional (based on the the Angular convention) recommends build:, chore:, ci:, docs:, style:,     refactor:, perf:, test:, and others.
+- footers other than BREAKING CHANGE: <description> may be provided and follow a convention similar to git trailer format.
+ 
+For more detailed reference to the above points, refer <a href="https://www.conventionalcommits.org/en/v1.0.0/">here</a>
+
+## 💥 Issues:
+For major changes, you are welcomed to open an issue  about what you would like to contribute. Enhancements will be appreciated.
+
+#### All the Best!🥇
```

Comment:
```suggestion
All the Best!🥇
```

Not sure this should be a header at all.

Reply:
I have corrected it. Thank you for pointing it out